rust/core-lib/src/editor.rs

```diff
@@ -1113,6 +1113,20 @@ impl Editor {
             RequestLines(LineRange { first, last }) => self.do_request_lines(first, last),
             Yank => self.yank(),
             Transpose => self.do_transpose(),
+            Click(MouseAction {line, column, flags, click_count} ) => {
+                // Deprecated (kept for client compatibility): should be removed in favor of do_gesture
+                println!("Usage of click is deprecated and should be replaced by gestures");
```

Comment:
you can't use `println` in xi-core because it prints to stdout, which is interpreted as an RPC by the client, which it won't be able to parse. To print to stderr, use the `eprintln!` macro.